example-config.yml

```diff
@@ -1,29 +1,51 @@
 # Date for filtered GTFS files
 date: 2019-09-10
 
-# Specify the extents to clip the graph to. These are for Bradford.
+# Specify the extents to clip the graph to. These are for Sheffield
 # Alternatively, you can provide a named extent (as defined in otp4gb/extents.yml) as a command line -b/--bounds option
 extents:
-  max_lon: -1.418634
-  min_lon: -2.084680
-  max_lat: 53.978646
-  min_lat: 53.605485
+  min_lon: -1.5279
+  max_lon: -1.4119
+  max_lat: 53.4416
+  min_lat: 53.3347
 
 # Specify the osm file to use
 osm_file: great-britain-latest.osm.pbf
 
 # Specify the list of GTFS files to process
 gtfs_files:
   - GBRail_GTFS.zip
+  - EM_GTFS.zip
   - Y_GTFS.zip
 
-# Options affecting the routing
-travel_time: 2019-09-10T08:30:00
-buffer_size: 100
+# Length of time to search before the travel time,
+# if not given then it is calculated based on route availability
+time_periods:
+  - name: AM
+    travel_time: 09:00
+    search_window_minutes: 60
+  - name: IP
+    travel_time: 14:00
+    search_window_minutes: 60
+  - name: PM
+    travel_time: 19:00
+    search_window_minutes: 60
+
 modes:
-  - BUS,WALK
-  - TRANSIT,WALK
-  - BICYCLE
-  - CAR
-  - WALK
-centroids: MSOACentroids_NorthEngland_LatLong.csv
```

Comment:
- centroids: 
^ Parameter is now missing from example-config.yml - has this moved elsewhere?

Reply:
Fixed in 130ec4ba50ac3934ee8259bb4439831a201ebd51